test/integration-tests/express/middleware-default-route-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const { expect } = require('chai');
+const Prometheus = require('prom-client');
+const supertest = require('supertest');
+
+let app, config;
+
+describe('when using express framework (default route)', function() {
```

Comment:
I made a new test server in this PR because the default route (wildcard `'*'`) assumes a paradigm where requests get handled by routes which don't call next(). 

In the existing tests, routes [always call next()](https://github.com/PayU/prometheus-api-metrics/blob/master/test/integration-tests/express/server/express-server.js#L30) and the test logic relies on this for the error cases.